core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -74,6 +74,22 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
   }
 
   /**
+   * Compute the population standard deviation of this RDD's elements.
+   * @since 2.1.0
```

Comment:
Ur, by the way, some scala code also use java since annotation.
- SparkSession.scala, StreamingQuery.scala, StreamingQueryManager.scala
Is it they designed for Java compatible?